extractor/src/test/java/org/schabi/newpipe/extractor/services/media_ccc/MediaCCCLiveStreamExtractorTest.java

```diff
@@ -0,0 +1,45 @@
+package org.schabi.newpipe.extractor.services.media_ccc;
+
+import org.junit.Assume;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.schabi.newpipe.downloader.DownloaderTestImpl;
+import org.schabi.newpipe.extractor.InfoItem;
+import org.schabi.newpipe.extractor.NewPipe;
+import org.schabi.newpipe.extractor.kiosk.KioskExtractor;
+import org.schabi.newpipe.extractor.stream.StreamExtractor;
+import org.schabi.newpipe.extractor.stream.StreamInfo;
+
+import java.util.List;
+
+import static org.schabi.newpipe.extractor.ServiceList.MediaCCC;
+
+public class MediaCCCLiveStreamExtractorTest {
+
+    private static KioskExtractor liveKiosk;
+    private static StreamExtractor extractor;
+
+    private static List<InfoItem> liveItems;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        NewPipe.init(DownloaderTestImpl.getInstance());
+        liveKiosk = MediaCCC.getKioskList().getExtractorById("live", null);
+        liveKiosk.fetchPage();
+        liveItems = liveKiosk.getInitialPage().getItems();
+        Assume.assumeFalse(
+                "Received an empty list of live streams. Skipping MediaCCCLiveStreamExtractorTest",
+                liveItems.isEmpty());
+    }
+
+    @Test
+    public void testRequiredStreamInfo() throws Exception {
```

Comment:
Shouldn't there be an assert?

Reply:
see the comment below

Reply:
A test that runs into an exception != A test that fails (through an ``AssertionError``).
See also https://rules.sonarsource.com/java/RSPEC-2699

``assertDoesNotThrow`` would be helpful here however as we aren't using JUnit 5 that's not available. (I think I should do a PR for that).

I would solve the problem like this:
```java
try {
    ...
} catch (final Exception e) {
    Assert.fail("An exception occurred. This should never happen.")
}
```

Reply:
Done